src/Traits/RequestTrait.php

```diff
@@ -255,7 +259,11 @@ protected function processResponse($service, $method, $response, $resultClass)
         );
 
         $msg = 'YDB ' . $service . ' ' . $method . ' (YDB_' . $statusCode . ' ' . $statusName . '): ' . $message;
-        if (isset(self::$ydbExceptions[$statusCode])) {
+        if (isset(self::$ydbExceptions[$statusCode]) && class_exists(self::$ydbExceptions[$statusCode])) {
```

Comment:
Need remove check class exist.
May be additional test function for check ok all exceptions.